CMakeLists.txt

```diff
@@ -139,7 +139,7 @@ else()
         add_definitions(-D_WINKD_)
     endif()
 
-    if(CMAKE_VERSION MATCHES "ReactOS" AND CMAKE_VERSION VERSION_LESS 3.3.0)
+    if(CMAKE_VERSION MATCHES "ReactOS")
```

Comment:
Make this `AND NOT CMAKE_VERSION VERSION_LESS 3.16.0`?
That way we don't have to 100% time the merge of this with the RosBE switch.

Reply:
Done.

Reply:
Are we keeping `cmake_minimum_required` where it is?